subsys/bluetooth/mesh/Kconfig

```diff
@@ -59,12 +59,16 @@ choice BT_MESH_ADV
 
 menuconfig BT_MESH_ADV_LEGACY
 	bool "Legacy advertising"
+	depends on BT_LL_SW_SPLIT
 	help
 	  Use legacy advertising commands for mesh sending. Legacy
-	  advertising is significantly slower than the extended advertising, but
-	  is supported by all controllers.
+	  advertising is significantly slower than the extended advertising.
 
-	  WARNING: The legacy advertiser can occasionally do more message
+	  WARNING: This feature is no longer supported in NCS. The
```

Comment:
Actually, this is not the truth. The feature is still supported. Please add technical reason here. I mean description of a real problem while it cannot be used.

Reply:
I mean, aren't we choosing to remove support for the feature?

For the last part, something like: 
> The legacy advertiser will not work with SDC, as attempting to start an advertisement during the scanner duty cycle will result in an error. In ZLL, a workaround is introduced to let Mesh advertise during scanning.

Reply:
AFAIK Legacy advertiser is still there.
I think we do not need to write here about ZLL workaround.

Reply:
But just because it is there does not mean that we will continue to support it? Isn't that the point of adding this dependency?

Reply:
Yep. What I want to see here is the real problem description (at least a pair of phrases) not a just message "this is not supported".

Reply:
Updated now